Ical.Net/DataTypes/CalDateTime.cs

```diff
@@ -303,68 +305,26 @@ public override int GetHashCode()
     }
 
     public static bool operator !=(CalDateTime? left, IDateTime? right)
-        => !(left == right);
-
-    /// <summary>
-    /// Subtracts a <see cref="TimeSpan"/> from the <see cref="CalDateTime"/>.
-    /// </summary>
-    /// <remarks>
-    /// This will also add a <see cref="IDateTime.Time"/> part that did not exist before the operation,
-    /// if the <see cref="TimeSpan"/> is not a multiple of 24 hours.
-    /// </remarks>
-    public static IDateTime operator -(CalDateTime left, TimeSpan right)
     {
-        var copy = left.Copy<CalDateTime>();
-        var newValue = copy.Value - right;
-        if (!copy.HasTime && (right.Ticks % TimeSpan.TicksPerDay) == 0)
-        {
-            copy._dateOnly = DateOnly.FromDateTime(newValue);
-        }
-        else
-        {
-            copy._dateOnly = DateOnly.FromDateTime(newValue);
-            copy._timeOnly = TruncateTimeToSeconds(newValue);
-        }
-        return copy;
+        return !(left == right);
     }
 
     /// <summary>
-    /// Adds a <see cref="TimeSpan"/> to the <see cref="CalDateTime"/>.
+    /// Creates a new instance of <see cref="CalDateTime"/> with <see langword="true"/> for <see cref="HasTime"/>
     /// </summary>
-    /// <remarks>
-    /// This will also add a <see cref="IDateTime.Time"/> part that did not exist before the operation,
-    /// if the <see cref="TimeSpan"/> is not a multiple of 24 hours.
-    /// </remarks>
-    public static IDateTime operator +(CalDateTime left, TimeSpan right)
+    public static implicit operator CalDateTime(DateTime left)
```

Comment:
I feel the implicit conversion operator is not too helpful anymore. Given our previous efforts of making working with `DateTime` more explicit, the implicit conversion seems to be counterproductive.

Reply:
tracked with #677